src/main/java/com/renatusnetwork/momentum/data/plots/PlotsDB.java

```diff
@@ -50,12 +50,23 @@ public static void addTrustedPlayer(int plotID, PlayerStats trustedPlayer)
                     "VALUES (?,?)", plotID, trustedPlayer.getUUID());
     }
 
+    public static void addTrustedPlayer(int plotID, String trustedPlayerUUID) {
+        DatabaseQueries.runAsyncQuery(
+                "INSERT INTO " + DatabaseManager.PLOTS_TRUSTED_PLAYERS_TABLE + " (plot_id, trusted_uuid) " +
+                        "VALUES (?,?)", plotID, trustedPlayerUUID);
+    }
+
     public static void removeTrustedPlayer(int plotID, PlayerStats trustedPlayer)
     {
```

Comment:
do we also need the PlayerStats removeTrusted and addTrusted in the DB methods? if not you can remove those too

Reply:
oops thought i removed those